### PR TITLE
Azure AD Docs improved

### PR DESCRIPTION
**Pull request by dorschw**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/41541

## Description
Setup instructions improved, added detailed permission information.

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md [hunk 1]
```diff
@@ -323,6 +325,7 @@ Retrieves the properties of a collection of riskDetection objects.
 ***
 Gets the risk history of a riskyUser resource.
 
+Permission required: `IdentityRiskyUser.Read.All`, `IdentityRiskyUser.ReadWrite.All`
```
Comment (dorschw) [inline Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md:328]:
```suggestion
Permissions required: `IdentityRiskyUser.Read.All`, `IdentityRiskyUser.ReadWrite.All`
```

File: Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ This integration was integrated and tested with the beta version of Azure Active
...
+    | Application ID | The ID of the managed application. Use one of the following: <ul><li>  **The Cortex XSOAR Azure application**. Use the application ID `4ffef4a4-601f-4393-a789-432f3f3b8470`, and fill in your subscription ID (from the Azure Portal). </li> <li> Add a **new Azure App Registration** in the Azure Portal, with the following permissions: <br>`IdentityRiskEvent.Read.All` <br> `User.Read` <br> `IdentityRiskyUser.ReadWrite.All` <br>Permission requirements of the different commands are detailed below. <br> When using a self-hosted app, make sure to `Allow public client flows`. This enables the Device-flow login mechanism, which this integration uses to authenticate), see instructions [here.](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication#device-code-flow)  <br> Set up the app to support multitenant accounts under `Authentication` on the Azure portal.  </li></ul> | True |
```
Comment (Itay4) [inline Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md:12]: we are abusing the app id desc here, please add it below somewhere

File: Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md [hunk 3]
```diff
@@ -114,6 +114,7 @@ There is no context output for this command.
 ***
 Retrieve the properties of a collection of riskDetection objects.
 
+Permission required: `IdentityRiskEvent.Read.All`
```
Comment (Itay4) [inline Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md:117]: show be according to our standard:
```
#### Required Permissions
```
see how its done in other integrations

File: Packs/AzureActiveDirectory/ReleaseNotes/1_1_3.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Azure Active Directory Identity Protection (Beta)
+- Updated the integration setup instructions.
+- Added detailed permission requirement information.
```
Comment (Itay4) [inline Packs/AzureActiveDirectory/ReleaseNotes/1_1_3.md:5]:
```suggestion
- Documentation and metadata improvements.
```

File: Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md [hunk 5]
```diff
@@ -1,6 +1,16 @@
 Gets information from the Azure Active Directory Identity Protection service.
 This integration was integrated and tested with the beta version of Azure Active Directory Identity Protection API.
 
+## Set up an Azure Application
```
Comment (Itay4) [inline Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md:4]:
```suggestion
## Authorization
```

Comment (Itay4) [inline Packs/AzureActiveDirectory/Integrations/AzureADIdentityProtection/README.md:4]: add `## Required Permissions` section

**Comment by okaufman34** [comment]
@Itay4  Doc review completed

**Pull request merged**